harness/determined/cli/version.py

```diff
@@ -6,20 +6,20 @@
 import termcolor
 from packaging import version
 
-import determined
-import determined.cli
+import determined as det
+from determined import cli
 from determined.cli import render
 from determined.common import api
 from determined.common.declarative_argparse import Cmd
 
 
-def get_version(host: str) -> Dict[str, Any]:
-    client_info = {"version": determined.__version__}
+def get_version(sess: api.BaseSession, host: str) -> Dict[str, Any]:
```

Comment:
I think including `host` might be an unnecessary footgun.

Reply:
yeah, this is a carryover from when I had Session leaving certain of its attributes as private.  It doesn't really make sense anymore.

Fixed.